templates/part.content.php

```diff
@@ -118,6 +118,12 @@
 			<div class="album-info-loader"></div>
 			<div class="album-info-content markdown-body"></div>
 		</div>
+		<!-- button for opening bin -->
```

Comment:
This button should be part of a separate PR. We don't have enough room to add a new button atm. It will be easier once buttons can be shown on 2-3 rows.